### PR TITLE
Support loading the URI from a file in subkey

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: bin/utils/subkey/src/main.rs [hunk 1]
```diff
@@ -244,6 +247,29 @@ fn main() {
...
+/// Get `URI` from CLI or prompt the user.
+///
+/// `URI` is extracted from `matches` by using `match_name`.
+///
+/// If the `URI` given as CLI argument is a file, the file content is taken as `URI`.
+/// If no `URI` is given to the CLI, the user is prompted for it.
+fn get_uri(match_name: &str, matches: &ArgMatches) -> String {
+	if let Some(uri) = matches.value_of(match_name) {
+		let file = PathBuf::from(uri);
+		if file.is_file() && file.exists() {
```
Comment (kianenigma) [inline bin/utils/subkey/src/main.rs]: is it needed to also do `.exists()`?

Comment (bkchr) [reply bin/utils/subkey/src/main.rs]: Ahh yeah, I have misread the documentation. This is not required.

**Comment by bkchr** [comment]
What you mean by prefer file over stdin? We don't do this?

**Comment by kianenigma** [comment]
Was just wondering where the PR is coming from, with what motivation. In other words, what is the advantage of reading from a file rather than from the command line's stdin?

(and I noted that I personally think file is better but wanted to know your side as well. )

**Comment by bkchr** [comment]
> Was just wondering where the PR is coming from, with what motivation. In other words, what is the advantage of reading from a file rather than from the command line's stdin?
> 
> (and I noted that I personally think file is better but wanted to know your side as well. )

Ahh okay :) The motivation came from using subkey :) I just don't wanted to copy my super secret uri into the clipboard :D 
Also I think it is a good pattern to support loading the URI from a file.

**Pull request merged**